test/extended/fixtures/Dockerfile-jenkins-test-new-plugin

```diff
@@ -0,0 +1,6 @@
+FROM openshift/jenkins-1-centos7
+USER root
+RUN touch /opt/openshift/configuration/plugins.txt
+COPY ./jpi /opt/openshift/plugins
+RUN ls -la /opt/openshift/plugins
```

Comment:
do you check the output of `ls`? can't it be one layer? (will speed up test run a little bit ;-)